### PR TITLE
[RFC] vim-patch:8.1.2233

### PR DESCRIPTION
**Pull request by kuuote**
#### vim-patch:8.1.2233: cannot get the Vim command line arguments

Problem:    Cannot get the Vim command line arguments.
Solution:   Add v:argv. (Dmitri Vereshchagin, closes vim/vim#1322)
https://github.com/vim/vim/commit/69bf634858a2a75f2984e42b1e4017bc529a040a

### CODE REVIEW

File: src/nvim/eval.c [hunk 1]
```diff
@@ -9106,6 +9107,27 @@ void set_vim_var_dict(const VimVarIndex idx, dict_T *const val)
   }
 }
 
+/*
+ * Set the v:argv list.
+ */
+  void
```
Comment (teto) [inline src/nvim/eval.c:9113]: remove the space and make sure the linter passes: the comments we use are  `///`

**Comment by janlazo** [comment]
`test_startup.vim` changes are missing. Need `check.vim` patches for the `:Check` commands.

**Comment by kuuote** [comment]
I knew I had to rewrite the test, so I wrote a new one.
Would it be better to delete it?

**Comment by janlazo** [comment]
It's fine to have both but Vim's test changes should be included in the patch unless they're N/A such as gui, screen dumps, terminal, job, channel, etc.

**Comment by kuuote** [comment]
OK, I'll be careful if I have an opportunity.

**Comment by tjdevries** [comment]
Hey @kuuote , PR looks good to me. Looks like you have a conflict w/ the test file that if you could fix up that'd be great and then we can merge I think.

**Comment by kuuote** [comment]
Thanks for reviewing, so I fixed it.

**Pull request merged**